stripe2qbo/api/routers/transaction_router.py

```diff
@@ -42,6 +40,4 @@ async def get_transaction_by_id(
         .filter(TransactionSyncORM.id == transaction_id)
         .first()
     )
-    if transaction is None:
-        raise HTTPException(status_code=404, detail="Transaction not found")
```

Comment:
here I think we should still throw an exception. Since the assumnptio when calling this endpoint is is that there does a transaction with that id